test/base.py

```diff
@@ -53,10 +62,6 @@ def setUpClass(self):  # noqa
         from app import app
         self.app = app
 
-    @classmethod
```

Comment:
This function gets called after the tests run.   This file is kinda messy as I look through it now... (I wrote it a *while* ago).   I'll clean things up, probably adding this back to do some `drop_collection()`s.